web-client/src/views/StartCase/Country.tsx

```diff
@@ -56,7 +56,7 @@ export const Country = connect(
               className="usa-radio__label"
               data-testid="international-country-btn"
```

Comment:
Should the data-testid change here as well?

Reply:
Never mind...i was looking at old commit.

Reply:
not a bad catch though!